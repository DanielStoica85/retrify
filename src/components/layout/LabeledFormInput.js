import React from 'react';
import { FormGroup, Label, Input } from 'reactstrap';

const LabeledFormInput = props => {
    const {
        labelText,
        labelFor,
        inputType,
        inputName,
        inputId,
        inputPlaceholder,
        handleChange,
        value
    } = props;
    return (
        <FormGroup>
            <Label for={labelFor}>{labelText}</Label>
            <Input
                type={inputType}
                name={inputName}
                id={inputId}
                placeholder={inputPlaceholder}
                onChange={handleChange}
                value={value}
            />
        </FormGroup>
    );
};

export default LabeledFormInput;
