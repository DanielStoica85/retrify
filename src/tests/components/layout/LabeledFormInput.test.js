import React from 'react';
import { shallow } from 'enzyme';
import LabeledFormInput from '../../../components/layout/LabeledFormInput';

describe('SignedInLinks component', () => {
    it('should render correctly', () => {
        const component = shallow(<LabeledFormInput />);
        expect(component).toMatchSnapshot();
    });
});
