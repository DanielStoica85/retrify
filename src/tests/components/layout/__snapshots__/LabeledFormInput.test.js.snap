// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedInLinks component should render correctly 1`] = `
<FormGroup
  tag="div"
>
  <Label
    tag="label"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  />
  <Input
    type="text"
  />
</FormGroup>
`;
